kubernetes-manifests/loadgenerator.yaml

```diff
@@ -35,7 +35,7 @@ spec:
         image: loadgenerator
         env:
         - name: FRONTEND_ADDR
-          value: "frontend:80"
+          value: "http://frontend:80"
```

Comment:
Hi Nicholas, could you give some more details on why you added this? This default value, `frontend`, is a Kubernetes service name. it is intended to be reached inside the cluster, and expands out to the kubedns name `frontend.<namespaces>.svc.cluster.local`. I'm not sure what adding an `http` is intended to do here.